app/bootstrap.php

```diff
@@ -160,5 +162,13 @@
         }
     }
 
+    foreach ((array) $config['extensions'] as $extensionClass) {
+        if (is_string($extensionClass)) {
+            $app['extensions']->add(new $extensionClass());
+        } else {
+            $app['extensions']->add($extensionClass);
+        }
+    }
+
```

Comment:
extensions services shouldn't be invoked here, extend the service. Maybe with an if check to see if there are any extensions to add. 

Also you should do `is_a` check for string and `instanceof` check for object, like we do for `services`, `resources` and `app`.